kolibri/plugins/registry.py

```diff
@@ -48,14 +67,19 @@ def initialize():
             try:
                 plugin_module = importlib.import_module(app + ".kolibri_plugin")
                 logger.debug("Loaded kolibri plugin: {}".format(app))
+                # Load a list of all class types in module
+                all_classes = [cls for cls in plugin_module.__dict__.values() if isinstance(cls, type)]
+                # Filter the list to only match the ones that belong to the module
+                # and not the ones that have been imported
+                all_classes = filter(lambda x: plugin_module.__package__ + ".kolibri_plugin" == x.__module__, all_classes)
```

Comment:
Yup, that's what the interwebs say :) I tried playing around with it, and it seems to work :)